### PR TITLE
Prepare for releasing 6.5.0

### PR DESCRIPTION
**Pull request by j-rivero**
# 🎈 Release

Preparation for 6.5.0 release.

Comparison to 6.4.0: https://github.com/ignitionrobotics/ign-gazebo/compare/ignition-gazebo6_6.4.0...prepare_6_5_0

## Checklist
- [x] Asked team if this is a good time for a release
- [x] There are no changes to be ported from the previous major version
- [x] Updated changelog
- [ ] Updated migration guide (as needed)
- [x] Bumped version in ign-gazebo6-release 

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -2,6 +2,79 @@
...
+    * [Pull request #1229](https://github.com/ignitionrobotics/ign-gazebo/pull/1229)
+
+1. Add project() call to examples
+    * [Pull request #1274](https://github.com/ignitionrobotics/ign-gazebo/pull/1274)
+
+1. Implement /server_control::stop
+    * [Pull request #1240](https://github.com/ignitionrobotics/ign-gazebo/pull/1240)
+
+1. 👩‍🌾 Make depth camera tests more robust (#897)
+    * [Pull request #897) (#1257](https://github.com/ignitionrobotics/ign-gazebo/pull/897) (#1257)
```
Comment (iche033) [inline Changelog.md:76]: this looks like a backport to citadel than forward ported. I think this should be released already and we can remove this entry

File: Changelog.md [hunk 2]
```diff
@@ -2,6 +2,79 @@
...
+    * [Pull request #1310](https://github.com/ignitionrobotics/ign-gazebo/pull/1310)
+
+1. Adds a `Link::SetLinearVelocity()` method
+    * [Pull request #1323](https://github.com/ignitionrobotics/ign-gazebo/pull/1323)
+
+1. Fix weird indentation in `Link.hh`
+    * [Pull request #1324](https://github.com/ignitionrobotics/ign-gazebo/pull/1324)
+
+1. fix buoyancy test by increasing tol
+    * [Pull request #fix buoyancy test by increasing tol](https://github.com/ignitionrobotics/ign-gazebo/pull/fix buoyancy test by increasing tol)
```
Comment (iche033) [inline Changelog.md:19]: this is from a commit done during a forward port in https://github.com/ignitionrobotics/ign-gazebo/pull/1322. We can remove this.

File: Changelog.md (hunk 1 above)
Comment (j-rivero) [reply Changelog.md]: [fcb02cb](https://github.com/ignitionrobotics/ign-gazebo/pull/1338/commits/fcb02cb56f8e50e743c70f03fcf43a52008de0d9)

File: Changelog.md (hunk 2 above)
Comment (j-rivero) [reply Changelog.md]: [fcb02cb](https://github.com/ignitionrobotics/ign-gazebo/pull/1338/commits/fcb02cb56f8e50e743c70f03fcf43a52008de0d9)

**Review by iche033 — APPROVED** [review]

**Pull request merged**